src/base/bittorrent/torrentinfo.h

```diff
@@ -48,14 +47,17 @@ namespace BitTorrent
     class InfoHash;
     struct TrackerEntry;
 
-    class TorrentInfo final : public AbstractFileStorage
+    class TorrentInfo final
```

Comment:
seems `final` has no proper meaning here, might consider removing it.

Reply:
> seems `final` has no proper meaning here, might consider removing it.

Removed.